tests/test_excluders/test_roiexcluder.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+
+from pkg_resources import require
+require("mock")
+from mock import MagicMock, patch, call
+
+from scanpointgenerator import ROIExcluder
+from scanpointgenerator.compat import np
+
+roi_patch_path = "scanpointgenerator.core.ROI"
+
+
+class TestCreateMask(unittest.TestCase):
+
+    def setUp(self):
+        self.r1 = MagicMock()
+        self.r2 = MagicMock()
+        self.e = ROIExcluder([self.r1, self.r2], ["x", "y"])
+
+    def test_mask_points_called_on_rois(self):
+        x_points = [1, 2, 3, 4]
+        y_points = [10, 10, 20, 20]
+        self.r1.mask_points.return_value = \
+            np.array([True, False, False, True])
+        self.r2.mask_points.return_value = \
+            np.array([False, False, True, True])
+        expected_mask = np.array([True, False, True, True])
+
+        mask = self.e.create_mask(x_points, y_points)
+
+        self.r1.mask_points.assert_called_once_with([x_points, y_points])
+        self.r2.mask_points.assert_called_once_with([x_points, y_points])
+        np.testing.assert_array_equal(expected_mask, mask)
```

Comment:
Normally use assertEquals(expected_mask.tolist(), mask.tolist()).
It doesn't matter much, but we might want these tests to run under Jython, and np.testing is unlikely to be implemented.

This is also the test that verifies the OR relation for the regions, but isn't part of the name. Not too concerned about that either.